config/develop/namespaced/s3-to-glue-lambda.yaml

```diff
@@ -0,0 +1,10 @@
+template:
```

Comment:
using sceptre to deploy is fine but it makes this lambda exclusive to this repo.  This seems like a pretty simple and generic lambda that can probably used by multiple projects?  If so i would consider making a separate repo for this lambda so it can be deployed from multiple repos.